.github/workflows/main.yml

```diff
@@ -1,23 +1,19 @@
-name: Ruby
+name: CI
 
-on:
-  push:
-    branches: [main]
-  pull_request:
-    branches: [main]
```

Comment:
I'm not sure how this was even running on other branches before 😅

Reply:
I think the syntax is, run this on pull requests to main branch